lms/templates/commerce/provider.underscore

```diff
@@ -1,13 +1,13 @@
 <div class="provider-wrapper">
   <div class="provider-info">
-    <%- edx.HtmlUtils.interpolateHtml(
-          gettext("You still need to visit the %(display_name)s website to complete the credit process."),
+    <%- HtmlUtils.interpolateHtml(
```

Comment:
For these two changes where you aren't mixing in HTML tags, just replace `HtmlUtils.interpolateHtml` with `StringUtils.interpolate`, otherwise you'd be double-escaping.